plugins/jobs-management/src/JobsListWidget/components/JobCard.tsx

```diff
@@ -11,6 +11,12 @@ const JobCard = observer(function JobCard({ job }: { job: NewJob }) {
           <strong>{'Name: '}</strong>
           {job.name}
         </Typography>
+        {job.statusMessage ? (
+          <Typography variant="body1">
+            <strong>{'Message: '}</strong>
```

Comment:
nit: don't have to use the curly for the plain-old-string on the strong line of the jsx